src/Pages/Api.js

```diff
@@ -29,4 +29,20 @@ const makeAuthenticatedApiCall = (method, path, data) => {
   });
 };
 
-export { makeUnAuthenticatedApiCall, makeAuthenticatedApiCall };
+const makeUpdateApiCall = (method, budget_id, data) => {
+  return axios({
+    method: method,
+    url: `https://atumaatu.herokuapp.com/v1/budgets/${budget_id}`,
```

Comment:
You could have just used the `makeAuthenticatedApiCall` and pass `budgets/2` as the path. You don't need to create a need method